app/components/headerBar.tsx

```diff
@@ -8,17 +9,49 @@ import SocialIcons from "app/components/socialIcons";
 import { internalUrls, externalUrls } from "app/lib/navigation/constants";
 
 export default function HeaderBar() {
+  const [expanded, setExpanded] = useState(false);
+
   return (
-    <Navbar fixed="top" collapseOnSelect expand="xl" bg="white">
+    <Navbar fixed="top" collapseOnSelect expand="xl" bg="white" expanded={expanded}>
       <Container>
         <Navbar.Brand href={internalUrls.home}>
           <AxoneLogo width="200" />
         </Navbar.Brand>
-        <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+        <Navbar.Toggle aria-controls="responsive-navbar-nav" onClick={() => setExpanded(expanded ? false : 'expanded')}>
```

Comment:
`setExpanded` function expect a boolean as argument and not a boolean union a string value. Why not:

```suggestion
        <Navbar.Toggle aria-controls="responsive-navbar-nav" onClick={() => setExpanded(!expanded)}>
```